compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/packaging/converters/PathConverter.java

```diff
@@ -123,7 +123,11 @@ public Path start() {
         }
     
         if (Files.isRegularFile(path)) {
-            return Stream.of(new FileSystemSourceInput(path));
+            return Stream.of(
```

Comment:
Shall we move this logic to getEntry name method?(like store relevant data in the FileSystemSourceInput class and do the calculation later?)

Reply:
Commit 4fc2d88ae0f28a6a5d1682f49e37e99ad038e089 addresses this suggestion. Through this commit I have simplified the logic of getting the relative paths of files. We don't have to explicitly handle the scenario `running as a file` as it's automatically handled with the simplified logic. 

Basically, path `ABC/foo/./` (or `ABC/foo/./././././`) is exactly same as path `ABC/foo` since `.` points to the CWD in the file system.